packages/terra-core-docs/src/terra-dev-site/doc/switch/AccessibilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,178 @@
+
+| Key/Sequence | Description |
+|---|---|
+|**Tab**| Moves focus on and off interactable elements. User expects elements to receive focus in the logical reading order of the page. |
+|**Enter**| Executes Switch action.  |
+
+## Support Compliance
+
+### Primary Criteria
+
+- [**1.1.1 Non-text Content**](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) - Supports 
+   - Terra provides the ability for information, structure, and relationships to be programmatically determined.
+   - Engineers must ensure any visual relationship with a switch and other content that conveys meaning must programmatically set associations to the content for visual relationships can be understood by screen reader users.
+   - Content creators must provide the meaningful switch text (the visual label) that communicates what users should expect when the switch is activated(on/off). Must also express to the engineer any visual relationships of a switch to other content for them to ensure engineers can programmatically convey the visual relationships.
+
+- [**2.1.1 Keyboard**](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) — Supports
+   - Terra provides the ability for all interactable elements to receive focus, be interacted with, and be acted upon via keyboard.
+   - Engineers must ensure that all interactable elements can receive focus, be interacted with, and are able to be acted upon using the keyboard.
+   - Content creators must ensure that all interactable elements can receive focus, be interacted with, and are able to be acted upon using the keyboard.
+
+- [**2.4.6 Headings and Labels**](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html) — Supports 
+   - Terra provides the ability to create labels that are descriptive of their purpose.
+   - Engineers must implement meaningful text for labels to accurately describe their purpose.
+   - Content creators are responsible for providing the unique label that describes the action the switch performs.
+
+- [**2.4.7 Focus Visible**](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) - Supports
+  - Terra provides the ability for actionable elements to receive visual focus.
+  - Engineers must ensure a visible keyboard focus is maintained on all interactive elements.
+  - Content creators must inform engineers that whenever switch receives focus, the focus must be visible and maintain existing Terra Keyboard Focus styles.
+
+- [**2.5.3 Label in Name**](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html) — Supports 
+  - Terra provides the ability for programmatic labels to match visual labels. 
+  - Engineers must ensure the programmatic label matches the visual labels when interactable elements are included with this component.
+  - Provide the engineer with the switch text (the visual labels) which must be unique and describe the purpose of the switch so it can be understood by the user out of context.
```

Comment:
```suggestion
  - Content creators must provide the engineer with the switch text (the visual labels) which must be unique and describe the purpose of the switch so it can be understood by a user out of context.
```